tests/test_dft_orca_task.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import pandas as pd
+from unittest.mock import patch, MagicMock
+from bluephos.tasks.dft_orca import dft_run, remove_second_row
+
+
+@pytest.fixture
+def setup_dataframe():
+    data = {
+        "ligand_identifier": ["H11A1L9"],
+        "xyz": [
+            """3
+H11A1L9
+H 0.0000 0.0000 0.0000
+H 0.0000 0.0000 0.9500
+O -0.9266 0.0000 -0.3333"""
+        ],
+    }
+    df = pd.DataFrame(data)
+    return df
+
+
+@patch("bluephos.tasks.dft_orca.tempfile.mkdtemp")
+@patch("bluephos.tasks.dft_orca.run_dft_calculation")
+@patch("bluephos.tasks.dft_orca.os.getenv")
+@patch("bluephos.tasks.dft_orca.extract")
+@patch("subprocess.run")
+def test_dft_run(mock_subprocess, mock_extract, mock_getenv, mock_run_dft_calculation, mock_mkdtemp, setup_dataframe):
+    mock_getenv.return_value = "mock/path/to/orca"
+    mock_mkdtemp.return_value = "/mock/temp/dir"
+    mock_run_dft_calculation.return_value = ("/mock/path/triplet_output.txt", "/mock/path/base_output.txt")
+    mock_extract.return_value = 0.1234
+    mock_subprocess.return_value = MagicMock()
```

Comment:
I wonder if you need all the mocks, or if we can let tempdir do its thing and just mock out the `run_orca_command` which is nicely isolated already. You can grab an output file and store it as test data which could be written out there, this would help test the extraction as well so it doesn't have to be mocked.

Reply:
Yes, have changed the test part to reduce the number of mocks and rely more on actual filesystem operations and make it closer to the real usage scenario